mac/help/start/install-keyman.md

```diff
@@ -3,7 +3,12 @@ title: Install Keyman for macOS
 ---
 
 *Keyman for macOS* is available for [download](https://keyman.com/mac/download) as a standard
-Apple Disk Image (.dmg) file. It should be installed according to the following instructions.
+Apple Disk Image (.dmg) file. You can also get the installer through Homebrew with the command:
+```
+brew install --cask keyman
+```
```

Comment:
If installing with Homebrew, then no need to install step-by-step, but will need to do additional security changes (I can't remember the differences off the top of my head).